### ISSUE TITLE
Is there a way to turn off re-render of the marks?

### ISSUE THREAD

**Issue by JFrogKevinz** [NightCatSama/vue-slider-component#371]
I'm implementing the slider with custom dot, tooltip and marks templates using respective slots but I have a performance issue when displaying a lot of marks (141).

I can see that each drag event is rendering all marks templates but in my case I didn't need to render them all the time.

I wanted to know if there is a way to turn off the re-drawing of marks templates when dragging or to put a debounce time to avoid performance issue.

Thank you.

**Comment by NightCatSama** [comment]
It seems that it should not be caused by repainting. 

It should be caused by the execution time of the script that gets the list of mark each time the component re-renders.

I will try to optimize it.

**Comment by NightCatSama** [comment]
In the latest version of chrome, if I didn't open the console, 200 marks would slide smoothly (I suspect that it is caused by frequent element updates on the Element).

**Comment by JFrogKevinz** [comment]
Thanks for your quick answer.
You were right. I didn't suspect the chrome devtools to be the source of the problem. When closing it the slider slide more smoothly.

I also removed all components in my view to be sure that anything else was eating resources (I implemented the slider in a 'Sandbox' view with a lot of others components before to use it in real views of my app). Now I can see that it's really smooth.

Nevertheless I think that it can be a good future feature to add a 'debouncer' props for repainting or a 'noUpdate' props to avoid completely the updates of the component to increase performance on slow devices (can be implemented with Object.freeze function).

Thank  you NightCatSama.
